dpnp/dpnp_array.py

```diff
@@ -108,6 +108,14 @@ def T(self):
         """View of the transposed array."""
         return self.transpose()
 
+    @property
+    def mT(self):
+        """View of the matrix transposed array."""
```

Comment:
Could you add examples from here: https://numpy.org/doc/stable/reference/generated/numpy.matrix.mT.html